src/components/UI/docs/Code.tsx

```diff
@@ -0,0 +1,46 @@
+// Libraries
+import { Code as ChakraCode, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+
+interface Props {
+  code: any;
+}
+
+export const Code: FC<Props> = ({ code }) => {
+  return (
+    !!code.inline ?
+      (
+        <Text
+          as='span'
+          background='code-bg'
+          fontFamily='"JetBrains Mono", monospace'
```

Comment:
`fontFamily`, `fontWeight`, `fontSize`, `lineHeight` and `letterSpacing` should be defined on theme's textStyles.